PicPi/UserPreferencesImpl.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  UserPreferencesImpl.swift
+//  PicPi
+//
+//  Created by Erick Sanchez on 12/23/20.
+//
+
+import Foundation
+
+func injectUserPreferences() -> UserPreferences {
+  return UserPreferencesImpl()
+}
+
+private class UserPreferencesImpl: UserPreferences {
+  private let piIPAddressKey = "Pi1IP"
+  private lazy var encoder = JSONEncoder()
+  private lazy var decoder = JSONDecoder()
```

Comment:
JSON to decode and encode Array of IPs?

Reply:
Yeah I was thinking we might start to store more than an array of strings like name, so might as well start using json

wdyt?